js/repl/CodeMirrorPanel.js

```diff
@@ -31,6 +39,9 @@ export default function CodeMirrorPanel(props: Props) {
           preserveScrollPosition={onChange == null}
           value={props.code}
         />
+        {options.fileSize && (
+          <div className={styles.fileSize}> Size: {fileSize} </div>
```

Comment:
One final thought: I wonder if we could get rid of the "Size:" prefix entirely? Given the units (eg KB, MB) it seems kind of unnecessary.) This would make the overlay even smaller, which would be nice.

```js
<div className={styles.fileSize}>{fileSize} </div>
```